workflow/util/util.go

```diff
@@ -810,7 +810,7 @@ func retryWorkflow(ctx context.Context, kubeClient kubernetes.Interface, hydrato
 			newNode := node.DeepCopy()
 			newNode.Phase = wfv1.NodeRunning
 			newNode.Message = ""
-			newNode.StartedAt = metav1.Time{Time: time.Now().UTC()}
```

Comment:
This seems wrong to me, the started time should be the time the pod was started, which is not now, or 0. I don't think this should have changed.

Reply:
If we don't change this, timeout/activeDeadline at template-level won't work correctly as the ttl controller is looking at the wrong timestamp for the node that's being retried.

Reply:
Is what way? I don't think we should hack the start time to fix anything, that'll create other bugs.

Reply:
Okay that makes sense. I'll close this now and defer to a better solution later that does not hack the start time.